torch/_dynamo/output_graph.py

```diff
@@ -533,7 +535,7 @@ def compile_subgraph(
         if stack_values:
             val_to_names[stack_values[-1]] = list()
         for k, v in tx.symbolic_locals.items():
-            if isinstance(v.source, LocalSource) and v.source.name() == k:
+            if isinstance(v.source, LocalSource) and v.source.local_name == k:
```

Comment:
We can't use name() because it references "G" or "L" while `k, v in tx.symbolic_locals.items()` knows it by its local_name

If you do not do this, you get spurious args in your graph. Let me know if you would like an example.